lib/rules/require-default-export.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * @author ItMaga
+ * See LICENSE file in root directory for full license.
+ */
+'use strict'
+
+const utils = require('../utils')
+
+module.exports = {
+  meta: {
+    type: 'problem',
+    docs: {
+      description: 'require components to be the default export',
+      categories: undefined,
+      url: 'https://eslint.vuejs.org/rules/require-default-export.html'
+    },
+    fixable: null,
+    schema: [],
+    messages: {
+      mustDefaultExport: 'The component must be the default export.'
```

Comment:
Now the message is not really fitting for when there is no export at all. I think we either need different error messages, or a single refined one:

```suggestion
      mustDefaultExport: 'Script block should have component as default export.'
```

or:

1. If there is no export at all and no component (`defineComponent(…)` or `Vue.component(…)` or similar) in a `.vue` file: `Missing default export.`
2. If there is a component: `Component must be the default export.`

Reply:
Thank you for the clarification. Fixed